tutorials/README.txt

```diff
@@ -1,3 +1,3 @@
 Tutorials
 =========
-This page contains the tutorials about TVM.
+This page contains the tutorials about TVM. For background knowledge of deep learning, please refer to the interactive book `Dive into Deep Learning <https://d2i.ai>`__, the `Deep Learning book <http://www.deeplearningbook.org/>`__, or `Stanford CS231n course <http://cs231n.stanford.edu/>`__.
```

Comment:
typo: https://d2l.ai

Reply:
We just updated the url to this one. It may take a while to take effect